lambda/dataRegisterUtils.js

```diff
@@ -3,9 +3,10 @@ const { logger } = require('./logger');
 
 const DATA_REGISTRY_URL = process.env.DATA_REGISTRY_URL;
 const DATA_REGISTER_NAME_API_KEY = process.env.DATA_REGISTER_NAME_API_KEY;
+const ESTABLISHED_STATE = 'established'
```

Comment:
`;` needed here and below

Reply:
fixed in [bb370dc](https://github.com/bcgov/parks-reso-api/pull/333/commits/bb370dc36dc2353f8985e9f00ae58b62539a7bc8)